tests/unit/sceneUpdateTests.cpp

```diff
@@ -0,0 +1,73 @@
+#include "catch.hpp"
+
+#include "yaml-cpp/yaml.h"
+#include "scene/sceneLoader.h"
+#include "style/style.h"
+#include "scene/scene.h"
+#include "platform.h"
+#include "tangram.h"
+
+using YAML::Node;
+using namespace Tangram;
+
+const char* path = "scene.yaml";
```

Comment:
Relying on the name and contents of our testing scene file is probably not a good idea. If we go and refactor or rename `scene.yaml` then these tests will all fail even though the code they're meant to test is unchanged. I'd recommend just using some cutsom YAML, it doesn't even have to be a complete scene specification.